subsys/power/policy/policy_residency_stm32.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <kernel.h>
+#include "pm_policy.h"
+
+#include <stm32l4xx_ll_lptim.h>
```

Comment:
Is that still required ?
If yes, this won't work with other series than L4

Reply:
also removing references to LPTIM clock